frontend/components/organisms/labels/LabelCreationForm.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <base-card
     :disabled="!valid"
-    title="Create Label"
-    agree-text="Create"
-    cancel-text="Cancel"
+    :title="$t('projects.createLabel')"
```

Comment:
Incorrect title:
![image](https://user-images.githubusercontent.com/6737785/94093352-f7218380-fe57-11ea-8e17-4a2be6324207.png)

Reply:
Addressed in https://github.com/doccano/doccano/pull/976/commits/aa2679edd15b629335e74a10781123373b0e7880

Reply:
Wrong. `labels.createLabel` is correct.

Reply:
My mistake. Fixed in https://github.com/doccano/doccano/pull/976/commits/5ea290cfa8b9247d4c6b6a24fb4acc22c4bc450d